src/Illuminate/Support/Str.php

```diff
@@ -234,6 +234,19 @@ public static function charAt($subject, $index)
         return mb_substr($subject, $index, 1);
     }
 
+    /**
+     * Given a multibyte string, return an array of its characters.
+     *
+     * @param  string  $string  The string to split into characters or chunks.
+     * @param  int  $length  If specified, each element of the returned array will be composed of multiple characters instead of a single character.
+     * @param  string|null  $encoding  The character encoding (default: null, which uses the internal encoding).
+     * @return array returns an array of strings.
+     */
+    public static function chunk(string $string, int $length = 1, ?string $encoding = null)
```

Comment:
According to `Str::covertCase`, default `$encoding` value should be `'UTF-8'`

Reply:
default value changed